include/aspect/utilities.h

```diff
@@ -379,21 +379,41 @@ namespace aspect
      * Note the required format of the input data: The first lines may contain
      * any number of comments if they begin with '#', but one of these lines
      * needs to contain the number of grid points in each dimension as for
-     * example '# POINTS: 3 3'. The order of the columns has to be
+     * example '# POINTS: 3 3'. The comments can optionally be followed by a
+     * single uncommented line containing the names of the data columns.
```

Comment:
uncommented = "commented out"? Maybe just say "a single line starting with '#'" to avoid the ambiguity